CHANGELOG.md

```diff
@@ -48,8 +50,15 @@ New modules
 
   Relation.Binary.Construct.Closure.Equivalence.Properties
   Relation.Binary.Rewriting
+
+  Relation.Binary.Properties.Setoid
   ```
 
+* ``Data/Bin.agda`` and ``Data.Bin/*.agda``  of lib-1.0 are removed,
```

Comment:
Please could you add an explanation why they have been replaced (i.e. that the new representation is easier to use because...)

Reply:
Done.